problem_builder/tasks.py

```diff
@@ -108,4 +106,60 @@ def scan_for_blocks(block):
         "report_filename": filename,
         "start_timestamp": start_timestamp,
         "generation_time_s": generation_time_s,
+        "display_data": [] if len(rows) == 1 else rows[1:]
```

Comment:
If this is used in a course that has dozens of problems and thousands of students, large data sets are easily possible. I think you should add a maximum size limit here. If there are >1,000 rows, it's probably quite inefficient to send them via celery to the LMS to the browser and display them using JavaScript. In such a case, we should just show a preview of the data and get them to download the CSV in order to see the whole thing.